tests/cpp17/test_cpp17.cpp

```diff
@@ -38,6 +39,138 @@ namespace cpp11 {
 #include "../optional_scalars_generated.h"
 }  // namespace cpp11
 
+using ::cpp17::MyGame::Example::Monster;
+using ::cpp17::MyGame::Example::Vec3;
+
+/*******************************************************************************
+** Build some FB objects.
+*******************************************************************************/
+Vec3 BuildVec3() {
+  using ::cpp17::MyGame::Example::Color;
+  using ::cpp17::MyGame::Example::Test;
+  return Vec3(
```

Comment:
Better to use `{ ... }` initialization bracers here.
The **x**, **y**, and **z** are `float` so `f` suffix is desired.

Reply:
Done+Done